test/fixtures/manifest-file

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
We need another file with a specific name to avoid modifying other tests or having multiple `Plug.Static` instances.